tests/test_cli.py

```diff
@@ -437,7 +445,11 @@ def __init__(self, mock_data):
     def get(self, resource, params=None):
         mock_requests = self.mock_data[resource]
         key = _params_to_url(params)
-        objects = mock_requests.pop(key)
+        try:
+            objects = mock_requests.pop(key)
+        except KeyError:
+            print(mock_requests.keys())
```

Comment:
nit: if this is intentional could consider adding a comment explaining why it's being printed out. print statements usually trigger a warning in my brain.

Reply:
not intentional, I'll remove it. Thanks

Reply:
67f0624